iothub/service/src/JobClient/JobClient.cs

```diff
@@ -32,6 +32,10 @@ public static JobClient CreateFromConnectionString(string connectionString)
         /// <returns> A JobClient instance. </returns>
         public static JobClient CreateFromConnectionString(string connectionString, HttpTransportSettings transportSettings)
         {
+#if NET451
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls;
```

Comment:
Pretty strange that there is no common point of code for security protocol between jobclient, registry manager, (anything else on service client) and you have to duplicate code